### PR TITLE
enable importas linter

### PR DESCRIPTION
**Pull request by k15r**
**Description**

Changes proposed in this pull request:

- enable importas linter
- fix remaining package names

**Related issue(s)**

### CODE REVIEW

File: .golangci.yaml [hunk 1]
```diff
@@ -105,11 +104,130 @@ linters-settings:
     custom-order: true
     skip-generated: true
   importas:
-    no-unaliased: true
+    no-unaliased: false
     no-extra-aliases: true
+    ## kubernetes APIs get the prefix `k`. As an example check `kcorev1`
```
Comment (marcobebway) [inline .golangci.yaml:109]: +1

File: .golangci.yaml [hunk 2]
```diff
@@ -105,11 +104,130 @@ linters-settings:
...
+      - pkg: k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset/fake
+        alias: kapiclientsetfake
+      - pkg: k8s.io/apimachinery/pkg/api/errors
+        alias: kerrors
+      - pkg: k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1
+        alias: kapiextensionsv1
+      - pkg: github.com/kyma-project/eventing-manager/testing
+        alias: eventingtesting
+      - pkg: k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset
+        alias: kapiclientset
```
Comment (marcobebway) [inline .golangci.yaml:149]: How about we use the alias `kapixclientset`? I think `apix` reads better for `apiextensions`.

File: .golangci.yaml [hunk 3]
```diff
@@ -105,11 +104,130 @@ linters-settings:
...
+      - pkg: sigs.k8s.io/controller-runtime
+        alias: ctrl
```
Comment (marcobebway) [inline .golangci.yaml:151]: How about the alias `kctrl` or `skctrl`? For `sigs.k8s`

File: .golangci.yaml (hunk 2 above)
Comment (k15r) [reply .golangci.yaml:149]: sure, works for me

File: .golangci.yaml (hunk 3 above)
Comment (k15r) [reply .golangci.yaml:151]: i personally would prefer `kctrl`.

Comment (marcobebway) [reply .golangci.yaml:151]: `kctrl` sounds good

**Pull request merged**